src/pykeen/models/unimodal/rotate.py

```diff
@@ -68,43 +98,12 @@ def __init__(
             preferred_device=preferred_device,
             random_seed=random_seed,
             regularizer=regularizer,
+            entity_initializer=xavier_uniform_,
+            relation_initializer=init_phases,
```

Comment:
should `init_phases` be an in-place function?

Reply:
Possible implementation:

```python
def init_phases_(x: torch.Tensor):
    """Initialize with random phases in place using :func:`init_phases`."""
    x.data = init_phases(x)
    return x
```

Reply:
Or, we can make sure that `reset_parameters` of `Embedding` does this wrapping for all init methods.

Reply:
```python
def reset_parameters():
    x.data = self.initializer(x.data)
```